pkg/users/client.go

```diff
@@ -238,6 +238,44 @@ type ListAuthFactorsResponse struct {
 	ListMetadata common.ListMetadata `json:"listMetadata"`
 }
 
+type InviteObject struct {
+	Object         string `json:"object"`
+	ID             string `json:"id"`
+	Email          string `json:"email"`
+	State          string `json:"state"`
```

Comment:
enumerate the different `state`s?
* pending
* accepted
* expired
* revoked

(not sure how to do that in go, hopefully we have some existing examples to follow...)

Reply:
No worries - easy enough, I'll add these :)